src/lincity-ng/MainLincity.cpp

```diff
@@ -113,6 +117,16 @@ void saveCityNG( std::string newFilename ){
         GameView* gv = getGameView();
         if( gv ){ gv->writeOrigin(); }
         save_city(const_cast<char*>( newFilename.c_str() ) );
+
+#ifdef EMSCRIPTEN
+    EM_ASM({
+        FS.syncfs(false, function(err) {
+            if (err) {
+                console.error("Error saving save data to web browser's IndexedDB storage: " + err);
+            }
+        });
+    });
```

Comment:
This should be indented with 4 more spaces to match up with the preceding lines.